src/main/java/dinkplugin/notifiers/DeathNotifier.java

```diff
@@ -242,6 +240,26 @@ private Template buildMessage(String killer, long losePrice, boolean pk, boolean
         return builder.build();
     }
 
+    private boolean shouldNotifyExceptionalDangerousDeath(ExceptionalDeath death) {
```

Comment:
This function is specifically for use in places where there are two types of deaths. 
1. A death where your character just dies, which is handled by on actor death.
2. A wipe of the full raid

Number 2 is what's being checked for here, making sure we don't fire for both events

Is this a good understanding of the function & its use?

Reply:
yep! if the safe death was allowed to fire, it would've happened in (1), so we shouldn't duplicate in (2). but if safe death was not fired and (2) occurs, it's a dangerous death, so we should fire notification now